internal/visitors/visitor_func.go

```diff
@@ -42,18 +41,32 @@ func (v funcVisitor) Visit(n ast.Node) ast.Visitor {
 	case *ast.FuncType:
 		// Should not need to declare any non-local definitions in the type params
 		// if node.TypeParams != nil {
-		// 	Walk(v, node.TypeParams)
-		// }
-
-		// Should not need to declare any non-local definitions in the params
-		// if node.Params != nil {
-		// 	Walk(v, node.Params)
+		// 	ast.Walk(v, node.TypeParams)
```

Comment:
Why is this commented code being modified but not being fully deleted? I'm confused...

Reply:
Im gonna either have a separate PR for type parameters or tie it into this PR as well